hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -2,19 +2,22 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 { name: "usbdev",
-  clock_primary: "clk_fixed",
-  other_clock_list: [ "clk_48MHz" ]
+  clock_primary: "clk_i",
+  other_clock_list: [ "clk_usb_48mhz_i" ]
+  reset_primary: "rst_ni",
+  other_reset_list: [ "rst_usb_48mhz_ni" ]
   bus_device: "tlul",
   bus_host: "none",
   available_inout_list: [
-    { name: "usb_dp", desc: "USB data D+" }
-    { name: "usb_dn", desc: "USB data D-" }
+    #{ name: "d", desc: "USB data differential" }
```

Comment:
`#` ?

Reply:
As far as I understood, we currently do not have support for differential signals in the pinmux/padctl. Thus, I did not want topgen to generate the I/Os. Instead I currently manually instantiate `usbdev` and feed in constant 0 on the differential data input port (see also your next comment).